src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.resx

```diff
@@ -122,7 +122,7 @@
   </data>
   <data name="ConfigCommandExamples" xml:space="preserve">
     <value>nuget config -Set HTTP_PROXY=http://127.0.0.1 -Set HTTP_PROXY.USER=domain\user
-nuget.config HTTP_PROXY</value>
+nuget config HTTP_PROXY</value>
```

Comment:
You should make sure the generated file gets changed as well. 

So either do a build or make this change in VS.